lib/YuiRestClient/Widget/ComboBox.pm

```diff
@@ -39,4 +39,12 @@ sub value {
     return $self->property('value');
 }
 
+# When combobox is enabled it does not have 'enabled' property. Only in case it is disabled, the property appears
+# and equals to 'false'.
+sub is_enabled {
+    my ($self) = @_;
+    my $is_enabled = $self->property('enabled');
+    return !defined $is_enabled && $is_enabled == 0;
```

Comment:
I guess you wanted to negate the whole condition, should not it be `!defined $is_enabled || $is_enabled != 0`?  And does the property return integer? I thought it was `true/false`, as per our ruby code:
```
      def enabled?
        enabled_prop = property(:enabled)
        enabled_prop.nil? || enabled_prop == true
      end
```

Reply:
Regarding first note you're right. Changed.

Regarding the second, it returns 'Perlish' `true/false`, as our JSON library converts them to '1/0':

This is what is shown if print parsed Hash with Dumper:
```
'hstretch' => bless( do{\(my $o = 1)}, 'JSON::PP::Boolean' ),
'enabled' => bless( do{\(my $o = 0)}, 'JSON::PP::Boolean' ),
```

Reply:
It sounds like we should then use perlish way to assert if it's true or false. I believe in combination with the `defined` check we don't have to check it against value. Otherwise, we can just compare against `1/0` as it guarantees that value is defined. I would prefer first way, so our code still works in case of changes in `JSON::PP::Boolean`

Reply:
> I believe in combination with the defined check we don't have to check it against value.
This check I added only for the case if `enabled` property is defined and equals to true. For this combobox it is not needed, but I'm not sure that there is no such elements, that has `enabled: true` property.

> Otherwise, we can just compare against 1/0 as it guarantees that value is defined. 
It doesn't, because for this particular case, when combobox is enabled it does not show 'enabled' property at all (at least for 'Filesystem' combobox, I'm not sure that it is not shown for other comboboxes with `enabled: true`). If you'll compare undefined value against '1/0', you'll get `Use of uninitialized value $is_enabled in numeric eq (==) `.

Reply:
AFAIR when it's disabled, property is defined in 100% cases, which is enough for this check.